src/lib/apis/orders/fetch-orders.ts

```
@@ -0,0 +1,32 @@
+        });
+
+        if (!res.ok) {
+            // HTTP error
+            throw new Error(`HTTP error! status: ${res.status}`);
+        }
+
+        const data: OrdersResponse = await res.json();
+        return data;
+    } catch (error) {
+        // Handle network errors or JSON parse errors
+        console.error("Failed to fetch orders:", error);
```

ESLint

[error] Unexpected console statement.

(no-console)